src/ocean/task/TaskPool.d

```diff
@@ -175,12 +180,16 @@ class TaskPool ( TaskT : Task ) : ObjectPool!(Task)
             item.terminationHook({
                 --count;
                 if ((count == 0) && task.suspended())
+                {
                     task.resume();
+                }
             });
         }
 
         if (count > 0)
+        {
```

Comment:
Well I made the comment for the `if` statement defined above and not for this one :)
And I still think it was the right thing to do, wasn't it?
```D
if ((count == 0) && task.suspended())
    task.resume();
```